project/SotaBuild.scala

```diff
@@ -89,6 +91,18 @@ object SotaBuild extends Build {
       testOptions in RandomTests += Tests.Argument(TestFrameworks.ScalaTest, "-n", "RandomTest"),
       parallelExecution in Test := true,
       dockerExposedPorts := Seq(8081),
+      dockerCommands := dockerCommands.value.filterNot {
```

Comment:
Ah I see.

Is there a way we can avoid getting a remote file to do this? Can we add that file locally and add it to the docker image? Would simplify these commands a bit and would be better as we don't fully control that file, if it's part of the deployment it should be part of our repo... 

We could then just add a file and include `bin/sota-resolver` in that file and use that as an entrypoint.